.github/workflows/integration-tests.yml

```diff
@@ -0,0 +1,20 @@
+---
+name:  Dependency Test 🔢
+
+on:
+  schedule:
+    - cron: '45 3 * * *'
```

Comment:
@pawelru The idea is to run the workflow nightly. Is this ok?

Reply:
Up to you guys. Those are long workflows so probably doesn't make much sense to execute on every push.